src/main/features/admin/v2/routes/dashboard.ts

```diff
@@ -10,14 +10,17 @@ import { AuthOptions } from 'request'
 class Renderer {
 
   static render ( res: express.Response ) {
-
     Promise.all ( [
-      FeesClient.searchFees ( 'draft', null ) /* TODO: Pass this user as author for draft fees */,
-      FeesClient.searchFees ( 'pending_approval', null )
+      FeesClient.searchFees('draft', null, null, null),
+      FeesClient.searchFees('pending_approval', null, null, null),
+      res.locals.user.allInfo.roles.indexOf('freg-approver') !== -1 ?
```

Comment:
There is a built in method available in user for this check n case you want to use. 
=> user.isInRoles('freg-approver')